### ISSUE TITLE
can the train.softmax.py only be used for generating embedding or can we do the prediction also

### ISSUE THREAD

**Issue by priyakansal** [davidsandberg/facenet#864]
Hi all,

I have built a face recognition. The process is as follows:
1) Used mtcnn_align.py to align my own images.
2) trained a classifier using classifier.py, in this I used the pretrained model weights to generate the embeddings.
3) recognize the face---- use weight of pretrained model to generate the embedding of test image, then .pkl file to recognize the face.

my question is ... is there any way that I can avoid the usage of .pkl file as classifier. Can the train.softmax.py only be used for generating embedding or can we do the prediction also.
